Doc/library/socket.rst

```diff
@@ -704,6 +704,13 @@ Constants
 
    .. versionadded:: 3.12
 
+.. data:: SHUT_RD
+          SHUT_WR
+          SHUT_RDWR
+
+   These constants are used by the :meth:`shutdown` method of socket objects.
```

Comment:
This seems to have lost the hyperlink. Maybe “~socket.socket.shutdown” would do it. See similar instances above, e.g. under SIO_RCVALL.

Reply:
Let me fix it.